src/components/messages/Message/Message.scss

```diff
@@ -48,6 +48,9 @@
     padding: 0.4em 0.8em;
     will-change: border-radius;
     transition: border-radius ease 0.2s;
+    @media only screen and (max-width: 700px) {
+      max-width: 80%;
+    }
```

Comment:
We like to go with mobile first, so if it should be `max-width: 80%` on mobile, then the default should be `max-width: 80%` and then have a media query `(min-width: 700px)`.

Reply:
```suggestion
    max-width: 80%;
    @media only screen and (min-width: 700px) {
        max-width: unset;
    }
```

Reply:
Ah alright! Sounds great :)
Will keep that in mind in future as well 🙌